src/Microsoft.ML.SamplesUtils/ConsoleUtils.cs

```diff
@@ -23,6 +23,18 @@ public static void PrintMetrics(BinaryClassificationMetrics metrics)
             Console.WriteLine($"Positive Recall: {metrics.PositiveRecall:F2}");
         }
 
+        /// <summary>
+        /// Pretty-print CalibratedBinaryClassificationMetrics objects.
+        /// </summary>
+        /// <param name="metrics">Calibrated binary classification metrics.</param>
```

Comment:
>binary classification metrics [](start = 45, length = 29)

use a see tag so it can be discovered by the tooling.

Reply:
done

---
In reply to: [259453147](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259453147) [](ancestors = 259453147)